src/pages/Editor/Classes/ClassComponent.tsx

```diff
@@ -77,31 +45,11 @@ const ClassComponent = ({ classes = [], fetchClasses }: classesProps) => {
               <div key={uuidv4()}>
                 <StyledClassItemsContainer>
                   <StyledClassItem>
```

Comment:
Hmm I think it would be better to do it like this
```typescript
const ClassItem = (description:string) => {
    return (
        <StyledClassItem>
          <ClassworkIcon /> {description}
         </StyledClassItem>
    )
}
{
   (innerElement.description) && (
     <ClassItem description={innerElement.description} />
   )
}
```